### DISCUSSION TITLE
How to add profile in DevDisplay?

### DISCUSSION THREAD

**Discussion by linamikou** [codeaashu/DevDisplay#382] [Q&A] [answered]
Give me step by step way to add profile in DevDisplay

**Comment by codeaashu** [comment, marked as answer]
> Give me step by step way to add profile in DevDisplay

<h3 align="center">Thank you for your interest in contributing to our open-source project!</h3>
<h2 align="center">The easiest way to contribute on Open Source - Hacktoberfest  &  GSSOC</h2>

<h1 align="center">Add your profile on DevDisplay <img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Telegram-Animated-Emojis/main/People/Technologist.webp" alt="Technologist" width="25" height="25" /></h1>

![Screenshot (1348)](https://github.com/user-attachments/assets/1a044709-bbe0-422b-891e-bd76d1ccdc13)

<h3 align="center">Steps to Add Your Profile Data <img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Hand%20gestures/Backhand%20Index%20Pointing%20Down.png" alt="Backhand Index Pointing Down" width="25" height="25" /></h3>

1. **Fork the repository:** To create a copy of the repository in your GitHub account, click on the "Fork" button in the top right corner of the project repository page.

2. **Navigate** to the **`public/data`** folder in your project directory.

3. **Create a new JSON file** named **`your_github_username.json`** (replace your_github_username with your actual GitHub username). Open the file you just created.

4. **Add** the following JSON object, replacing the placeholder values with your own details:

   ```json
   {
     "name": "Your Name",
     "location": "Your Location",
     "bio": "Your Bio should be 20-30 words not more than that",
     "avatar": "https://github.com/your-github-username.png",
     "portfolio": "Your Portfolio URL or Github URL",
     "skills": ["Your Skill 1", "Your Skill 2", "..."],
     "social": {
       "GitHub": "https://github.com/your-github-username",
       "Twitter": "https://twitter.com/your-twitter-username",
       "LinkedIn": "https://www.linkedin.com/in/your-linkedin-username"
     }
   }
   ```

5. **Save** the **`your_github_username.json`** file.

6. **Navigate** to the **`src`** folder in your project directory. Open the **`ProfilesList.json`** file.

7. **Add your JSON filename** (your_github_username.json) to the array of filenames in the ProfileList.json file, like this:

   ```json
   ["filename1.json", "filename2.json", "filename3.json", "your_github_username.json"]
   ```

8. **Save** the **`ProfileLists.json`** file.

9. **Contribute** Follow this 👇🏻:
![How to contribute](https://github.com/user-attachments/assets/731ac3c7-4878-476f-bd4a-fd4a39ae2849)


10. **Wait for review and merge:** Wait for the project maintainers to review and merge your changes.


<h3 align="center">Once your changes are merged <img src="https://raw.githubusercontent.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/master/Emojis/Activities/Party%20Popper.png" alt="Party Popper" width="25" height="25" /> Your profile will be displayed on the project's website.</h3><hr>
